### PR TITLE
style: some styling changes for wrapper

### PR DESCRIPTION
**Pull request by developerstew**
## Description

Brief description of changes made e.g. bug fix for broken Ajax container.

## Related tickets

Any Jira tickets related to the changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] I have lint my changes locally.
- [ ] I have formatted my changes locally.
- [ ] I have updated any relevant documentation.
- [ ] I have added tests to cover my changes.
- [ ] I have checked that all new and existing tests passed.
- [ ] I have checked to ensure there aren't other open Pull Requests for the same update/change.

### CODE REVIEW

**Review by mantonides — APPROVED** [review]

**Review by MikeyAlmighty — DISMISSED** [review]

File: src/components/detail-panel/index.js [hunk 1]
```diff
@@ -92,8 +92,15 @@ DetailPanel.Url = ({className, children, ...rest}) => (
 	</a>
 );
 
-DetailPanel.Wrapper = ({className, children, ...rest}) => (
-	<div className={cn(styles.Wrapper, className)} {...rest}>
+DetailPanel.Wrapper = ({className, isHorisontal, children, ...rest}) => (
+	<div
+		className={cn(
+			{[mixins.FlexStartSpaced]: isHorisontal},
```
Comment (tiaanduplessis) [inline src/components/detail-panel/index.js:98]: Misspelledling. Should be `isHorizontal`

**Review by tiaanduplessis — CHANGES_REQUESTED** [review]

**Review by Zazzzles — APPROVED** [review]

File: src/components/detail-panel/index.js (hunk 1 above)
Comment (mantonides) [reply src/components/detail-panel/index.js:98]: The props and variable being used is different and still not correct. Just make sure of the spelling.

**Review by mantonides — DISMISSED** [review]

File: src/components/detail-panel/index.js (hunk 1 above)
Comment (developerstew) [reply src/components/detail-panel/index.js]: For some reason the change is not showing here,but it is done when you look at Files changed

File: src/components/detail-panel/index.module.css [hunk 2]
```diff
@@ -54,10 +54,6 @@
   color: var(--color-dark3-60);
 }
 
-.Wrapper {
```
Comment (mantonides) [inline src/components/detail-panel/index.module.css:57]: If you remove the class please remove it at the component as well.

**Review by mantonides — CHANGES_REQUESTED** [review]

File: src/components/detail-panel/index.module.css (hunk 2 above)
Comment (developerstew) [reply src/components/detail-panel/index.module.css:57]: Removed

**Review by mantonides — APPROVED** [review]

**Review by Zazzzles — APPROVED** [review]

**Pull request merged**